src/utils/post_files_to_sandbox.py

```diff
@@ -9,8 +9,7 @@
 }
 
 response_one = requests.get(
```

Comment:
This file either need to be updated or removed, since the authentication for sandbox and qa changed

Reply:
removed the file